test/integration/peggy-e2e-test.md

```diff
@@ -12,46 +12,70 @@ sifnoded start
 
 1. open new console
 cd smart-contracts
-ebrelayer init tcp://localhost:26657 ws://localhost:7545/ 0x30753E4A8aad7F8597332E813735Def5dD395028 user1 --chain-id=sifchain
+
+ebrelayer init tcp://localhost:26657 ws://localhost:7545/ \
+0xEcFcaB0A285d3380E488A39B4BB21e777f8A4EaC \ # BridgeRegistry contract address
+akasha <akasha's mnemonic> --chain-id=sifchain
```

Comment:
This command needs to be as copy pastable as possible, so please provide the command in full with the mnemonic on a single line, otherwise it can confuse users.

Reply:
The only reason why I start creating this PR and the only goal is to create this documented command. Because initial one was completely not workable (and I came to the conclusion that this .md file is deprecated and not supported anymore). 
As you mentioned correctly, making it commented breaks "copy pastability". The are possible options:

1. To close this PR.
2. To provide some way where it would be possible to clearly comment the command (or to do some dynamic parameters resolution as alternative) AND at the same time to make it copy pastable. It could be some bash/shell tricks or any other programming language script from scratch (eg Python). And it should be done with team guidance regarding style, otherwise it may be not understood because of possible complexity.